public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -273,6 +283,17 @@ export class PanelModel {
         return srcValue;
       }
     });
+
+    this.fieldConfig = _.mergeWith(
+      {},
+      plugin.fieldConfigDefaults,
+      this.fieldConfig || {},
+      (objValue: any, srcValue: any): any => {
+        if (_.isArray(srcValue)) {
+          return srcValue;
+        }
+      }
+    );
```

Comment:
merging the overrides array, how does that work?

Reply:
uuuups, it probably doesn't ;)

Reply:
Updated the logic and added test for the behaviour